### ISSUE TITLE
Whitelist setting for auto skip accept invite step on room joining invites

### ISSUE THREAD

**Issue by MurzNN** [vector-im/element-meta#1573]

**Comment by tgr** [comment]
Yeah, this is a big pain point for some bridges (e.g. Slack) - if I wanted to switch over to our bridged Matrix/Element instance, I'd have some 200 invites to get through to be able to communicate with people... The hack from https://github.com/vector-im/element-meta/issues/1573#issuecomment-1543863464 doesn't seem to be working anymore, either.
